include/Entities/Laser.hpp

```diff
@@ -0,0 +1,47 @@
+//
+// Created by Ruoying Hao on 10/31/17.
+//
+
+#ifndef LASER_HPP
+#define LASER_HPP
+
+#include "Entities/Entity.hpp"
+
+class Laser : public Entity{
+
+protected:
+    sf::Vector2f pos_;
+    bool hacked_;
```

Comment:
will Enemy need to be able to face in a direction? If we can share several things between Laser and Enemy then maybe it is worth making a superclass for both of them

Reply:
Yes Enemy would need to face toward a direction. I wonder if Character would also need to do that, even it is controlled by the user. If so we may just implement it in Entity.